### PR TITLE
Half-finished version of job description

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

**Comment by humitos** [comment]
🤖 rendered post: https://readthedocs-blog--153.org.readthedocs.build/job-product-developer/

File: hiring/intro.rst [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-We are a small team of 7, and we're looking to add 1-2 people in 2022,
+We are a small team of less than 10, and we're looking to add 1-2 people in 2022,
 and we'd love for you to be one of them.
 
 About the company
 -----------------
 
 Read the Docs is a large `open source project <https://github.com/readthedocs/readthedocs.org>`_ providing documentation building and hosting.
-You can think of it a `continuous integration <https://en.wikipedia.org/wiki/Continuous_integration>`_ for docs.
+You can think of it a `continuous integration <https://en.wikipedia.org/wiki/Continuous_integration>`_ for software docs.
```
Comment (humitos) [inline hiring/intro.rst:9]:
```suggestion
You can think of it as a `continuous integration <https://en.wikipedia.org/wiki/Continuous_integration>`_ for software docs.
```

File: hiring/details.rst [hunk 2]
```diff
@@ -53,3 +35,18 @@ We are able to provide a supportive and very flexible work environment,
 with a strong sense of mission.
 All of the members of our team have worked with us for more than 2 years,
```
Comment (humitos) [inline hiring/details.rst:36]: Maybe this can be rephrased as "Most of the members of our team have worked with us for more than 4 years" (4 people from 6, that's it) because we hired people last year 😄

**Review by humitos — COMMENTED** [review]
The job description sound _very accurate_ to me. However, I'm seeing that as a downside because we are asking for a python developer + business/marketing person + support + operations. It sounds like _a lot to me_ and a little scary.

I know that we do a lot of that on a daily basis, but I thought the goal was to split responsibilities and reduce the context switch we need to jump from "coding a feature" to "writing a marketing blog post" so we can delegate one of those tasks to the right person. Not only the context switch will be reduced, but also the content/topics to study which will make our days smoother and our work to be better.

In my opinion, it would be good to make more emphasis on the business/marketing side than on the Python side. Our team is currently better at coding than at marketing, and I think we could grow a lot more by putting more effort into the marketing side. More marketing will bring more users, more users will bring more ideas/feature requests, more feature requests with well-handled product discussions between coders, marketers, and business people will bring us to $1M, I think. On the other hand, more developers without enough new users or without clear paths about what's the next feature to implement will make them be blocked.

Knowing Python is always good, but I wouldn't say that's the main knowledge required for this position. I'd like to have a person that knows how to write technical and marketing content than another Python developer that finds it hard to contact customers to sell a plan upgrade.

To summarize, I found the blog post not communicating exactly what _I thought_ the job was about (or the person we need) and also hard to digest for a person looking for a job. As a Python developer reading for this job position, I would not be sure if the position is for me or not because it seems I'm not the audience for this position.

Unfortunately, I don't have good suggestions about how to re-write the blog post or some paragraphs, but I tried to give as much feedback as I could relate to the whole post.

**Review by agjohnson — APPROVED** [review]
This post does a great job of describing the role in a practical way :+1: 

We discussed already how we might want to improve this post, and I could see us adding a bit more personality into the post as well. But I do think this does a good job for now of describing the position.

I included a couple additions to help round out the description more. I don't think it needs to be too accurate of a description though

File: hiring/intro.rst [hunk 3]
```diff
@@ -1,21 +1,18 @@
...
-We are a small team of 7, and we're looking to add 1-2 people in 2022,
+We are a small team of less than 10, and we're looking to add 1-2 people in 2022,
 and we'd love for you to be one of them.
 
 About the company
 -----------------
 
 Read the Docs is a large `open source project <https://github.com/readthedocs/readthedocs.org>`_ providing documentation building and hosting.
-You can think of it a `continuous integration <https://en.wikipedia.org/wiki/Continuous_integration>`_ for docs.
+You can think of it as a `continuous integration <https://en.wikipedia.org/wiki/Continuous_integration>`_ for software docs.
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply hiring/intro.rst:9]: Yea, that was a typo

File: job-product-developer.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+but hope for someone who is excited to learn how to do them.
+
+Who would be a good fit?
+------------------------
+
+Any of these people would be a good fit:
+
+* Someone who has built, documented, and blogged about a feature in an open source project
+* Someone who has advocated for a new feature being built at their current company, and then built it
+* Someone who has worked to get a feature that they built used by more people via a blog post and improved docs
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply job-product-developer.rst:45]: This is more of a marketing aspect, getting more usage by users.
